### PR TITLE
Refactor demucs get_module to return an nn.Module

### PR DESCRIPTION
**Pull request by jansel**
This also removes a cpu=>gpu copy from the timed section of the code, so it will change the score.

### CODE REVIEW

File: torchbenchmark/models/maml/meta.py [hunk 1]
```diff
@@ -161,8 +161,6 @@ def finetunning(self, x_spt, y_spt, x_qry, y_qry):
         :param y_qry:   [querysz]
         :return:
         """
-        assert len(x_spt.shape) == 4
```
Comment (jansel) [inline torchbenchmark/models/maml/meta.py:164]: Note this is an unrelated change to make this module `fx.symbolic_trace`

**Review by wconstab — APPROVED** [review]

**Comment by wconstab** [comment]
@Krovatkin I am thinking you can get your PR in first and then try to rebase/merge this one on top, rather than have to change yours.

**Comment by Krovatkin** [comment]
@wconstab Yes please! Lemme quickly address your comments and merge it
